packages/SwingSet/src/vats/network/network.js

```diff
@@ -237,11 +237,7 @@ export function makeNetworkProtocol(protocolHandler) {
   const bind = async localAddr => {
     // Check if we are underspecified (ends in slash)
     const underspecified = localAddr.endsWith(ENDPOINT_SEPARATOR);
-    const whileUnderspecified = asyncGenerate(() => ({
-      done: !underspecified,
-      value: null,
-    }));
```

Comment:
I think my new code is equiv to the old code. But both now seem non-sensical because `underspecified` is `const`. I don't see how this could have ever worked.

Reply:
There's a `break` in the loop.

Reply:
Aha! Thanks. Resolving.